docs/tutorials/schemaless-indexing.md

```diff
@@ -0,0 +1,194 @@
+# Place the cursor on schema strictness
+
+Quickwit let's you place define a document mapping that is  it is possible to use quickwit with a very strict mapping, in a pure schemaless manner, and anywhere in between. In other words, it is up to you to define how strict the schema should be handled.
```

Comment:
something's wrong here
 
Quickwit let's you place define a document mapping that is it is possible to use quickwit with a very strict mapping